src/components/menus/PlanMenu.svelte

```diff
@@ -138,6 +145,10 @@
       <MenuItem on:click={viewSnapshotHistory}>
         <div class="column-name">View Snapshot History</div>
       </MenuItem>
+      <MenuDivider />
```

Comment:
Curious why we wouldn't just have this be a normal menu button?
<img width="188" alt="image" src="https://github.com/user-attachments/assets/cd1370f1-00c0-442c-a4e8-69d0ac70fd6a">

Reply:
Yeah. That's a good question. I'll defer to @lklyne

Reply:
Updated!
![Screenshot 2024-08-06 at 4 27 33 PM](https://github.com/user-attachments/assets/ca4e2e63-6d92-4744-a2d8-4baa443c963f)